cli/internal/terraform/terraform_test.go

```diff
@@ -24,8 +24,13 @@ import (
 	"github.com/spf13/afero"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
 )
 
+func TestMain(m *testing.M) {
+	goleak.VerifyTestMain(m)
+}
```

Comment:
why do we need this? Should we add this to our testing principles that tests of concurrent code should always check this?

Reply:
We should add this to every package that contains tests.
Even if we don't use concurrent code directly, it sometimes uncovers concurrency bugs in code that we import.

> Should we add this to our testing principles...

I think this should be added to our testing principles.

Reply:
create a new ticket